test/test_mark.py

```diff
@@ -61,6 +61,21 @@ def test_status_mark_good_non_bootslot(rauc_no_service):
     assert exitcode == 1
 
 
+@have_grub
+def test_status_mark_bad_other(rauc_dbus_service_with_system_abc):
```

Comment:
Also add a test case for the `mark-good booted` with enabled `prevent-fallback`. You can use the `system` fixture to update the config and write it again from the test case.